src/adservice/src/main/java/hipstershop/AdServiceClient.java

```diff
@@ -97,14 +97,13 @@ public void getAds(String contextKey) {
     }
   }
 
-  private static int getPortOrDefaultFromArgs(String[] args, int index, int defaultPort) {
-    int portNumber = defaultPort;
-    if (index < args.length) {
+  private static int getPortOrDefaultFromArgs(String[] args) {
+    int portNumber = 9555;
+    if (2 < args.length) {
```

Comment:
does this change the behavior at all? I think we support $PORT env var to be able to also set the port number.

Reply:
No it doesn't, the port can still be dynamically set. This just moves the static parameters inside the function.